src/worker/tasks/send-transaction-worker.ts

```diff
@@ -199,17 +206,25 @@ const _sendUserOp = async (
       }
     }
 
+    const transactions = queuedTransaction.batchOperations
+      ? queuedTransaction.batchOperations.map((op) => ({
+          ...op,
+          chain,
+          client: thirdwebClient,
+        }))
+      : [
+          {
+            client: thirdwebClient,
+            chain,
+            ...queuedTransaction,
+            ...overrides,
+            to: getChecksumAddress(toAddress),
+          },
+        ];
```

Comment:
We're explicitly ignoring `overrides` in the batch. That said, I recall discussion that gas/gasFee overrides for userOps don't make sense and we should remove them anyway. We should pick one or the other: this code does it both ways (overrides for single userOp, no overrides for batch userOps).

Reply:
_Non-blocking comment (maybe cleanup for later)_

Single vs batch userOp flows should overlap as much as possible: there should be minimal difference other than the array of transactions being submitted.

That said I think all the "single vs batch userOp" logic should be in one place.

```
// pseudocode
let transactions = []
if (batchOperations) {
  for (op of batchOperations) {
    assert(op.to)
    transactions.push({ ...op, chain, client })
  }
} else {
  transactions.push({ to, data, value, chain, client })
}

// ...the rest of the code doesn't care about batchOperations
```

Reply:
> gas/gasFee overrides for userOps don't make sense

the current overrides don't work for userOps correct, but overrides in general can still be changed to use with userOps:

```
type UserOpGasOverrides = {
  callGasLimit?: bigint
  verificationGasLimit?: bigint
  preVerificationGas?: bigint
  maxFeePerGas?: bigint  
  maxPriorityFeePerGas?: bigint
}
```

even in the batch userop flow, there's only a single UserOp being constructed. And gas overrides are applied per UserOp, not per transaction inside the userop.